apps/web/src/app/user/my-claims/page.tsx

```diff
@@ -0,0 +1,235 @@
+import OrderListPriceItem from "~/app/_components/features/OrderListPriceItem";
+import { ProfileOptionLayout } from "~/app/_components/features/ProfileOptionLayout";
+
+const SalesStatusEnum = {
+	Paid: "Paid",
+	Prepared: "Prepared",
+	Shipped: "Shipped",
+	Delivered: "Delivered",
+} as const;
+
+type SalesStatus = (typeof SalesStatusEnum)[keyof typeof SalesStatusEnum];
+
+const DeliveryMethodEnum = {
+	Address: "Address",
+	Meetup: "Meetup",
+} as const;
+
+type DeliveryMethod =
+	(typeof DeliveryMethodEnum)[keyof typeof DeliveryMethodEnum];
+
+const mockedOrders = [
+	{
+		date: "October 18",
+		items: [
+			{
+				id: "1",
+				productName: "productName",
+				buyerName: "buyer1_fullname",
+				status: SalesStatusEnum.Delivered as SalesStatus,
+				delivery: DeliveryMethodEnum.Address as DeliveryMethod,
+				price: 30,
+				claimed: false,
+			},
+			{
+				id: "2",
+				productName: "productName2",
+				buyerName: "buyer2_fullname",
+				status: SalesStatusEnum.Delivered as SalesStatus,
+				delivery: DeliveryMethodEnum.Meetup as DeliveryMethod,
+				price: 30,
+				claimed: false,
+			},
+		],
+	},
+	{
+		date: "September 20",
+		items: [
+			{
+				id: "3",
+				productName: "productName3",
+				buyerName: "buyer1_fullname",
+				status: SalesStatusEnum.Delivered as SalesStatus,
+				delivery: DeliveryMethodEnum.Address as DeliveryMethod,
+				price: 30,
+				claimed: true,
+			},
+			{
+				id: "4",
+				productName: "productName4",
+				buyerName: "buyer2_fullname",
+				status: SalesStatusEnum.Delivered as SalesStatus,
+				delivery: DeliveryMethodEnum.Meetup as DeliveryMethod,
+				price: 30,
+				claimed: true,
+			},
+		],
+	},
+];
+
+export default function MyClaims() {
+	const [OrdersToClaim, setOrdersToClaim] = useState(mockedOrders);
+	const [ClaimedOrders, setClaimedOrders] = useState(mockedOrders);
+	const [searchTerm, setSearchTerm] = useState("");
+	const [MoneyToClaim, setMoneyToClaim] = useState(0);
+	const [isFiltersModalOpen, setIsFiltersModalOpen] = useState(false);
+
+	const router = useRouter();
+
+	useEffect(() => {
+		const unclaimedOrders = mockedOrders
+			.map((orderGroup) => ({
+				...orderGroup,
+				items: orderGroup.items.filter((item) => !item.claimed),
+			}))
+			.filter((orderGroup) => orderGroup.items.length > 0);
+
+		setOrdersToClaim(unclaimedOrders);
+
+		const totalMoneyToClaim = unclaimedOrders.reduce((total, orderGroup) => {
+			return (
+				total +
+				orderGroup.items.reduce(
+					(groupTotal, item) => groupTotal + item.price,
+					0,
+				)
+			);
+		}, 0);
+
+		setMoneyToClaim(totalMoneyToClaim);
+
+		const claimedOrders = mockedOrders
+			.map((orderGroup) => ({
+				...orderGroup,
+				items: orderGroup.items.filter((item) => item.claimed),
+			}))
+			.filter((orderGroup) => orderGroup.items.length > 0);
+
+		setClaimedOrders(claimedOrders);
+	}, []);
+
+	const openFiltersModal = () => {
+		setIsFiltersModalOpen(true);
```

Comment:
This is doing nothing at the moment, let me know if it should do something

Reply:
@muguika do we need this filter here?

![image](https://github.com/user-attachments/assets/4e670dbe-b37a-451e-8816-58f5e8aea8c0)

Reply:
not vital, no!